douane.c

```diff
@@ -1193,7 +1194,11 @@ static int __init initialize_module(void)
   INIT_LIST_HEAD(&process_socket_inodes.list);
 
   // Hook to Netfilter
+#if LINUX_VERSION_CODE <= KERNEL_VERSION(4,12,14)
   nf_register_hook(&nfho_outgoing);
+#else
+  nf_register_net_hook(&init_net, &nfho_outgoing);
```

Comment:
So there are no changes to [the `static struct nf_hook_ops nfho_outgoing`](https://github.com/cynay/douane-dkms/blob/34c7f35059ba80026eae88226689aa4397027036/douane.c#L1174) used to initialise the hook?

Reply:
nf_hook_ops is the same as before (adding the .dev caused a freeze and is not needet because we filter on every device)